Info param doc (#42)
agama-project/agama-project.github.io: docs/user/boot_options/linuxrc_compatibility.md

---
sidebar_position: 6
---

# Linuxrc Compatibility

> Agama provides few options to make easier transition from linuxrc. They are not identical and works

Reviewer: ```suggestion
Agama provides a few options to make it easier the transition from linuxrc. They are not identical and work
```

> slightly differently, but for usual usage it should be sufficient.

Reviewer: ```suggestion
slightly differently, but for a regular usage they should be sufficient.
```

> Also agama is case sensitive and require exact parameter usage unlike linuxrc which allows

Reviewer: ```suggestion
Also Agama is case sensitive and requires the exact parameter unlike linuxrc which allows
```
